src/github.com/getlantern/flashlight/client/fronted.go

```diff
@@ -1,17 +1,5 @@
 package client
 
-import (
```

Comment:
I took out a bunch of unused fronted stuff while I was in here, which also meant taking out the unused server functionality in flashlight.